test/test_files/abl_sampling/abl_sampling.inp

```diff
@@ -54,6 +54,7 @@ ABL.surface_roughness_z0 = 0.15
 #.......................................#
 amr.n_cell              = 48 48 48    # Grid cells at coarsest AMRlevel
 amr.max_level           = 0           # Max AMR level in hierarchy
+amr.max_grid_size       = 8
```

Comment:
what is the significance of this change? to ensure that box checks matter?

Reply:
yes exacly. I want to make sure that the samplers are crossing boxes.